app/views/pipelines/_grouped_requests.html.erb

```diff
@@ -82,16 +80,11 @@
           <td><%= link_to(h(request_asset.display_name), receptacle_path(request.asset)) %></td>
           <%= render partial: "request_comments", locals: { request: request, comments_count: request_comments_count } %>
           <td><%=  link_to "#{request_asset.sti_type} #{request_asset.try(:human_barcode)}", receptacle_path(request.asset) %></td>
-          <% unless asset_comments_count.zero? -%>
-            <%= render partial: "asset_comments", locals: { request: request, comments_count: asset_comments_count } %>
-          <% else -%>
-            <td>&nbsp;</td>
-          <% end -%>
-          <td>&nbsp;</td>
+          <td></td>
```

Comment:
So comments all grouped together now and so don't need separate asset comments section?

Reply:
Yep, it seemed simpler. Otherwise I'd either need three columns, labware, receptacle and request, or else group the first two under asset and still keep the request column.